scripts/packaging/copyright

```diff
@@ -0,0 +1,556 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
Thank you, great improvements!

But we really cannot maintain several copyright files, we hardly can manage one. Simply modify doc/THIRD-PARTY-LICENSES or remove doc/THIRD-PARTY-LICENSES (after updating this file) but please **do not duplicate** copyright information. It is too much of a burden for anyone adding something with copyright. The copyright file of the debian branch was often forgotten to be updated.

Reply:
Maintaining the doc/THIRD-PARTY-LICENSES in licencecheck format, like suggested previously, wouldn't really work. `licensecheck` analyzes each file in the project and returns the copyright information of the file. The problem is that wildcards for files are not supported, so we would need to mention each file explicitly. 

It would be probably best if we maintain the doc/THIRD-PARTY-LICENSES in dep5 format.

Reply:
Yes, doc/THIRD-PARTY-LICENSES in dep5 format is perfectly fine.